cmake/templates/OpenCVConfig-version.cmake.in

```diff
@@ -8,7 +8,3 @@ if(PACKAGE_FIND_VERSION VERSION_EQUAL PACKAGE_VERSION)
   set(PACKAGE_VERSION_EXACT True)
   set(PACKAGE_VERSION_COMPATIBLE True)
 endif()
-
-if(PACKAGE_FIND_VERSION VERSION_LESS PACKAGE_VERSION)
```

Comment:
> I'm not sure, should we handle case of PACKAGE_FIND_VERSION_MAJOR=0 here or not?
I don't think so. If version is not requested and `PACKAGE_FIND_VERSION_MAJOR` is 0, than the expression will be false and `PACKAGE_VERSION_COMPATIBLE` will be false.